### DISCUSSION TITLE
How to uninstall properly

### DISCUSSION THREAD

**Discussion by flawrist** [sboesen/remotely-sync#52] [General]
Dear @sboesen, I'd like to uninstall this plugin. Is it enough to remove it via the standard procedure (Options/Community plugins/Uninstall) or do I need to do something else?

**Comment by sboesen** [comment]
Uninstalling the normal way should work! Note this won't delete the remote vault folder if you want to reclaim space.

Please reopen if you need further help or if this isn't working for you, thanks!
